packages/web/src/builders/package/package.impl.ts

```diff
@@ -166,21 +168,24 @@ export function run(
       } else {
         context.logger.info('Bundling...');
         return from(rollupOptions).pipe(
-          mergeScan(
-            (acc, options) =>
-              runRollup(options).pipe(
-                map(result => {
-                  return {
-                    success: acc.success && result.success
-                  };
-                })
-              ),
+          mergeMap(options => runRollup(options)),
+          scan(
+            (acc, result) => {
+              return {
+                success: acc.success && result.success
+              };
+            },
```

Comment:
I found an issue here with a race condition: if one of the later rollup bundles failed and the first one succeeded, it would still mark the bundle as a **success** 

solution was to separate it into a `mergeMap` followed by a `scan` --> this way we still keep the parallelism of the bundle generators, but we correctly reduce the final result

I made a stackblitz here to show the issue: https://stackblitz.com/edit/rxjs-cqwcxx